mars-sim-core/pom.xml

```diff
@@ -84,7 +84,13 @@
 			<artifactId>jocl-main</artifactId>
 			<version>2.5.0</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.mars-sim</groupId>
+            <artifactId>mars-sim-ui</artifactId>
+            <version>3.8.0</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

Comment:
Making mars-sim-core dependent upon mars-sim-ui could lead to a circular dependency.

Reply:
It will prevent the system building from clean. That is why the PR checks have failed